packages/cli/src/constants/constants.ts

```diff
@@ -15,6 +15,7 @@ export const API_APPLICATION_ME_URL = `${apiAddress}/v1/applications/me`;
 export const API_SWITCH_APPLICATION_FORMAT_URL = `${apiAddress}/v1/auth/applications/{applicationId}/switch`;
 export const API_NOTIFICATION_GROUP_URL = `${apiAddress}/v1/notification-groups`;
 export const API_NOTIFICATION_TEMPLATES_URL = `${apiAddress}/v1/notification-templates`;
+export const API_TRIGGER_URL = `${apiAddress}/v1/events/trigger`;
 
 // Client
-export const CLIENT_LOGIN_URL = 'http://localhost:4200/v1/auth/login';
+export const CLIENT_LOGIN_URL = 'http://localhost:4200/login';
```

Comment:
```suggestion
export const CLIENT_LOGIN_URL = 'http://localhost:4200/auth/login';
```

Reply:
Are you sure about it? Basically, I imitated github oauth redirect url CLIENT_SUCCESS_AUTH_REDIRECT, it is strange because I do not see such a route (http://localhost:4200/login) on the client however it is still working in the flow test.
Let me validate it again.